guides/common/modules/proc_changing_the_content_source_of_a_host.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Changing_the_Content_Source_of_a_Host{context}"]
+= Changing the Content Source of a Host
+
+A Content Source is a {SmartProxy} that a host consumes content from. Use this procedure to change the Content Source of a Host
```

Comment:
```suggestion
A content source is a {SmartProxy} that a host consumes content from. Use this procedure to change the content source of a host
```

There are a few terms that are unique terms related to the project and they require capitalization. However, all other terms are just common nouns. A content source is just a source of content, not a real product term. It's a very common mistake to capitalize all terms in a project, but not necessary.